packages/@sanity/desk-tool/src/panes/document/useDocumentIsReferenced.tsx

```diff
@@ -0,0 +1,13 @@
+import {createHookFromObservableFactory, getPublishedId} from '@sanity/base/_internal'
+import documentStore from 'part:@sanity/base/datastore/document'
+import {Observable} from 'rxjs'
+
+export const useDocumentsIsReferenced = createHookFromObservableFactory((documentId: string) => {
```

Comment:
I was a bit hasty in suggesting we create a separate hook for this - it looks like we probably want to reuse the [useReferringDocuments](https://github.com/sanity-io/sanity/blob/next/packages/@sanity/desk-tool/src/components/confirmDeleteDialog/useReferringDocuments.ts#L140) hook, since it also handles cross-project references.

Ideally we'd lift the hook out of `confirmDeleteDialog` since it would then be used in multiple locations.

Reply:
Makes sense - I'll delete the other hook and use `useReferringDocuments` instead 🙌